x-pack/plugins/observability/public/hooks/slo/use_fetch_slo_groups.ts

```diff
@@ -27,20 +28,22 @@ interface UseFetchSloGroupsResponse {
 export function useFetchSloGroups({
   page = 1,
   perPage = DEFAULT_SLO_GROUPS_PAGE_SIZE,
+  groupBy = 'ungrouped',
```

Comment:
Nit: I would prefer to see `ungrouped` renamed to `none`, just cause it's shorter but not a huge deal.

Reply:
Yep good point, let me change this in a follow up PR. I can change this as part of this [issue](url), which I plan to work next